app/models/concerns/votes_counter.rb

```diff
@@ -16,7 +16,13 @@ module VotesCounter
   end
 
   def votes_count
-    try(:total_votes) || votes.sum(:value)
+    try(:total_votes) || votes_alternative
+  end
+
+  def votes_alternative
+    votes.reduce(0) do |_ , v|
+      _.to_i + v.value
```

Comment:
There is really no need for the conversion to integer (`:to_i`). What do you think?
... and @andela-iamadi will **wince** if he sees these variable names :speak_no_evil: